scripts/Import-ChocoServerCertificate.ps1

```diff
@@ -0,0 +1,79 @@
+<#
+.SYNOPSIS
+Connects to the QDE server and adds its SSL certificate to the local certificate
+store.
+#>
+[CmdletBinding()]
+param(
+    # The hostname of the QDE server to retrieve the SSL certificate from.
+    [Parameter()]
+    [Alias('NexusServer', 'NexusUrl')]
+    [string]
+    $ComputerName = '{{hostname}}'
```

Comment:
```suggestion
$ComputerName = '{{hostname}}'
```

Does this need to change?

Reply:
Yes! I added the `-replace` here:
https://github.com/chocolatey/choco-quickstart-scripts/blob/5de0d1f57084ef9a6cae2a6b8c4ca16bc478330d/scripts/New-IISCertificateHost.ps1#L40